src/lib/API/api.js

```diff
@@ -143,6 +143,14 @@ class API {
     return this.client.post('/verify/sendotp', { user })
   }
 
+  /**
+   * `/verify/sendnewotp` post api call
+   * @param {String} mobile
+   */
+  sendNewOTP(mobile: String): AxiosPromise<any> {
```

Comment:
see server comment
do we really need new APIs? can't we use same api as in signup?

Reply:
@sirpy we can't, because there are such conditions in code like this - https://prnt.sc/pe9jbt. The /sendemail request has it too. And nothing will happen then

Reply:
so modify it to also send if incoming body.user.mobile!==user.mobile

Reply:
ok